src/app/editor/editor.component.ts

```diff
@@ -3,21 +3,10 @@ import { MatExpansionPanel } from '@angular/material/expansion';
 import { MatDrawerContainer } from '@angular/material/sidenav';
 import { Store } from '@ngxs/store';
 import * as FileSaver from 'file-saver';
-import {
-  CurrentLayer,
-  CurrentPage,
-  ResetUiSettings,
-  Toggle3D,
-  ToggleCodeEditor,
-  ToggleWireframe,
-  UiState,
-  ZoomIn,
-  ZoomOut
-} from 'src/app/core/state';
+import { CurrentLayer, CurrentPage, ResetUiSettings, Toggle3D, ToggleCodeEditor, ToggleWireframe, UiState, ZoomIn, ZoomOut } from 'src/app/core/state';
```

Comment:
Maybe make this relative instead of `src/app/core/state`

Reply:
we need to review the whole path mapping in an upcoming refactoring.